src/components/ContractFilter/styles.js

```diff
@@ -0,0 +1,105 @@
+import { createUseStyles } from "react-jss";
+
+const useStyles = createUseStyles(
+  (theme) => ({
+    contractFilter: {
+      backgroundColor: theme.backgroundSecondary,
+      padding: "17px 24px",
+      borderRadius: "30px 0 30px 30px",
+    },
+    inputsContainer: {
+      width: "100%",
+      display: "flex",
+      gap: 25,
+    },
+    leftInputsContainer: {
+      width: "100%",
+      display: "flex",
+      flexDirection: "column",
+      gap: 12,
+    },
+    rightInputsContainer: {
+      width: "100%",
+      display: "flex",
+      gap: 12,
+    },
+    rightInputsColumn: {
+      width: "100%",
+      display: "flex",
+      flexDirection: "column",
+      gap: 10,
+    },
+    leftInputGroup: {
+      width: "100%",
+    },
+    rightInputGroup: {
+      width: "100%",
+    },
+    label: {
+      fontSize: 14,
+      color: theme.primaryColor,
+    },
+    inputsRow: {
+      width: "100%",
+      display: "flex",
+      gap: 20,
+    },
+    inputWithIcon: {
+      position: "relative",
+    },
+    subLabel: {
+      fontSize: 8,
+      color: theme.textStrong,
+      position: "absolute",
+      top: "12%",
+      background: theme.backgroundSecondary,
+      paddingInline: 6,
+      left: 12,
+      transform: "translateY(-50%)",
+    },
+    coinIcon: {
+      position: "absolute",
+      top: "55%",
+      left: 12,
+      transform: "translateY(-50%)",
+    },
+    leftInput: {
+      backgroundColor: theme.backgroundBase,
+      width: "100%",
+      borderRadius: 6,
+      border: "none",
+      padding: 10,
+      marginTop: 6,
+    },
+    rightInput: {
+      backgroundColor: "transparent",
+      width: "100%",
+      borderRadius: 6,
```

Comment:
create global variables vor border radiuses, something like this
  --radius-1: 8px;
  --radius-2: 12px;
  --radius-3: 16px;
  --radius-4: 20px;
  you can check all border radiuses in template